pyth.py

```diff
@@ -1,3 +1,6 @@
 print('-' * 30)
 print("hello world!")
-print('-' * 30)
\ No newline at end of file
+print('-' * 30)
+# print a variable 
+a  = 3 
+print(f"a  =  {a}")
```

Comment:
try to use a string formating compatible with previous versions of python

Reply:
consider it done
```suggestion
a  = 3 
print("a = " + str(a))
```
```suggestion
a  = 3 
print("a = %s" %a)
```